modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -1003,6 +1005,11 @@ const FDSEntries = ({
 						label: Liferay.Language.get('delete'),
 						onClick: onDeleteClick,
 					},
+					{
+						href: permissionsURL,
```

Comment:
A thing that FDS can help to enforce is to show/hide this action (and the above ones) depending on item permissions. This is achieved by adding a `permissionKey` to the item so that FDS checks the existence of such key in the item `actions` json key.